group_vars/all.yml.sample

```diff
@@ -326,6 +326,8 @@ dummy:
 #radosgw_dns_s3website_name: your.subdomain.tld # subdomain used by radosgw for website bucket hosting.
 # Rados Gateway options
 #email_address: foo@bar.com
+#TODO: PROVIDE SUGGESTIONS FOR DIFFERENT CONFIGURATIONS
```

Comment:
Do not edit this file directly, if it's a rgw option then this should go into `roles/ceph-rgw/defaults/main.yml`, if it's global this goes into: roles/ceph-common/defaults/main.yml`. Then run `generate_group_vars_sample.sh`.